docs/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
It's best to use the lowest supported CPython for generating such deps since this results in a better chance of generating a more complete transitive dep tree as some of the deps may be conditional.

Reply:
Marking this thread as resolved.